### DISCUSSION TITLE
2022/04/30/%E6%9C%9F%E6%9C%9B%E9%95%BF%E5%BA%A6P1365,CF235B,P1654/

### DISCUSSION THREAD

**Discussion by giscus** [StarCried/StarCried.github.io#4] [Announcements]
# 2022/04/30/%E6%9C%9F%E6%9C%9B%E9%95%BF%E5%BA%A6P1365,CF235B,P1654/

StarCried 的狼窝，记录算法竞赛知识（OI, ACM）以及学习、工作、生活中的点点滴滴。 - StarCried - StarCried 的狼窝

https://starcried.github.io/2022/04/30/%E6%9C%9F%E6%9C%9B%E9%95%BF%E5%BA%A6P1365,CF235B,P1654/

**Comment by Rappy325** [comment]
來搓板子的我忽然碼起了代碼（
